curriculum/challenges/english/10-coding-interview-prep/rosetta-code/abundant-deficient-and-perfect-number-classifications.md

```diff
@@ -32,32 +32,38 @@ Implement a function that calculates how many of the integers from `1` to `20,00
 assert(typeof getDPA === 'function');
 ```
 
-`getDPA` should return an array.
+`getDPA(5000)` should return an array.
 
 ```js
-assert(Array.isArray(getDPA(100)));
+assert(Array.isArray(getDPA(5000)));
 ```
 
-`getDPA` return value should have a length of 3.
+`getDPA(5000)` return array should have a length of `3`.
 
 ```js
-assert(getDPA(100).length === 3);
+assert(getDPA(5000).length === 3);
 ```
 
-`getDPA(20000)` should equal [15043, 4, 4953]
+`getDPA(5000)` should return `[3758, 3, 1239]`.
 
 ```js
-assert.deepEqual(getDPA(20000), solution);
+assert.deepEqual(getDPA(5000), [3758, 3, 1239]);
 ```
 
-# --seed--
+`getDPA(10000)` should return `[7508, 4, 2488]`.
+
+```js
+assert.deepEqual(getDPA(10000), [7508, 4, 2488]);
+```
 
-## --after-user-code--
+`getDPA(20000)` should return `[15043, 4, 4953]`.
 
 ```js
-const solution = [15043, 4, 4953];
+assert.deepEqual(getDPA(20000), [15043, 4, 4953]);
 ```
```

Comment:
This has got to be one of the most confusing diff I have seen lol. Took me a minute to get it.